vars/essentialsTest.groovy

```diff
@@ -10,7 +11,9 @@ def call(Map params = [:]) {
 
         dir(baseDir) {
             def metadataPath = "${pwd()}/${metadataFile}"
-            metadata = readYaml(file: metadataPath)
+            def configData = readYaml(file: metadataPath)
```

Comment:
The variable was lacking a `def` and I have changed the name of it to avoid having a `metadata.metadata` in the code which may be misleading, and I like the idea of a `metadata`section in the yml file